lightning/src/ln/channelmanager.rs

```diff
@@ -10453,30 +10590,42 @@ This indicates a bug inside LDK. Please report this error at https://github.com/
 							// concerning this channel as it is safe to do so.
 							debug_assert!(matches!(err, ChannelError::Close(_)));
 							let mut chan = Channel::from(inbound_chan);
-							return Err(convert_channel_err!(self, peer_state, err, &mut chan).1);
+							return Err(self.convert_channel_err(
+								&mut peer_state.closed_channel_monitor_update_ids,
+								&mut peer_state.in_flight_monitor_updates,
```

Comment:
How is this more readable than the macro was? I mean its not really a big deal but I don't see how more parameters that used to be hidden makes this more readable.